README.md

```diff
@@ -24,15 +24,15 @@ Because I was looking for an excuse to build a standalone component and publish
 ### Demo
 ![img](demo.gif)
 
-Check it out [here](http://prakhar1989.github.io/react-tags/example/)
+Check it out [here](https://react-tag-input.stackblitz.io)
 
 ### Installation
 The preferred way of using the component is via NPM
 
 ```
 npm install --save react-tag-input
 ```
-It is, however, also available to be used separately (`dist/ReactTags.min.js`). If you prefer this method remember to include [ReactDND](https://github.com/gaearon/react-dnd) as a dependancy. Refer to the [demo](http://prakhar1989.github.io/react-tags/example/) to see how this works.
+It is, however, also available to be used separately (`dist/ReactTags.min.js`). If you prefer this method remember to include [ReactDND](https://github.com/gaearon/react-dnd) as a dependancy. Refer to the [example](https://stackblitz.com/edit/react-tag-input) to see how this works.
```

Comment:
Added the direct link to the editor here. Should we add it someplace as well or does this suffice?

Reply:
Should be enough :+1: